code/gimkuku/week1/kakao_92335.py

```diff
@@ -1 +1,40 @@
-    
\ No newline at end of file
+import math
+
+# 소수 판별 함수
+def is_prime_num(n):
+    # 1일때 예외 처리
+    if (n == 1) :
+        return False
+    
+    sqrt_n = math.sqrt(n)
+    sqrt_n = int(sqrt_n) 
+    for i in range(2,sqrt_n +1):
+        if n % i == 0:
+            return False
+    return True
```

Comment:
```suggestion
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True
```

이러면 `math` 모듈을 import할 필요가 없어집니다...!

Reply:
아 **0.5 하는 방법이 있었군용.. 훨간단하네여!